### PR TITLE
Product list - add product and filter components

### PR DESCRIPTION
**Pull request by ValesarroRS**
#### Trello board reference:

- [Trello Card #] 

- https://trello.com/c/Lv61kRo6

#### Description:

- Create product component for desktop and mobile resolution
- Add list of filters at the left section 

#### Reviewers:

- @diebarral 
- @prina92 
- @joaquincollado 
- @DJA89 
- @kevin-rodriguez 

#### Notes:

- This does not include filter functionality
- Pagination is also not included

#### Tasks:

- https://trello.com/c/Lv61kRo6

#### Risk:

- N/A

#### Preview:
Screenshot 2023-05-19 at 18 13 30
Screenshot 2023-05-19 at 18 13 38

### CODE REVIEW

File: src/components/Filters/index.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  const [minPrice, setMinPrice] = useState('');
+  const [maxPrice, setMaxPrice] = useState('');
+
+  const submitFilters = () => {
+    const filters = {
+      condition,
+      category,
+      price: { min: minPrice, max: maxPrice },
+    };
+    console.log(filters);
```
Comment (diebarral) [inline src/components/Filters/index.tsx:17]: Remove console.log

Comment (diebarral) [reply src/components/Filters/index.tsx:17]: Ah I guess this is a feature to implement in the near future, it's ok, leave it like this

File: src/components/Filters/index.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  };
+
+  return (
+    <div className='ml-8 flex w-64 flex-col'>
+      <h1 className='mb-4 text-xl font-bold'>Filters</h1>
+      <h2 className='mb-2 font-medium'>Condition</h2>
+      <div className='mb-4 flex flex-col'>
+        {['New', 'Used', 'Refurbished'].map((item, i) => (
```
Comment (diebarral) [inline src/components/Filters/index.tsx:25]: You could move this array to a constant

File: src/components/ProductListed/index.tsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import Heart from 'components/shared/Icons/Heart';
+import Button, { ButtonSize, ButtonVariants } from 'components/shared/Buttons';
+
+export enum ProductTypes {
+  Restored = 'Restored',
+  New = 'New',
+}
```
Comment (diebarral) [inline src/components/ProductListed/index.tsx:8]: Shouldn't these match to the strings in the array I commented above? Shouldn't "Used" be here?

**Review by joaquincollado — APPROVED** [review]

**Pull request merged**